src/main/java/sandbox/SqliteIndexedStorageUtility.java

```diff
@@ -193,8 +192,8 @@ public T getRecordForValue(String fieldName, Object value)
             if (!resultSet.next()) {
                 throw new NoSuchElementException();
             }
-            byte[] mBytes = resultSet.getBytes(org.commcare.modern.database.DatabaseHelper.DATA_COL);
-            return readFromBytes(mBytes);
+            byte[] mBytes = resultSet.getBytes(DatabaseHelper.DATA_COL);
```

Comment:
i don't have a sense at when in the process this gets called, but if `mBytes` can be big, may want to use a stream